pkgs/development/tools/unityhub/default.nix

```diff
@@ -2,8 +2,14 @@
 
 let
   version = "2.3.2";
-in appimageTools.wrapType2 rec {
+  src = fetchurl {
+    # mirror of https://public-cdn.cloud.unity3d.com/hub/prod/UnityHub.AppImage
+    url = "https://archive.org/download/unity-hub-${version}/UnityHub.AppImage";
```

Comment:
Why not use web.archive.org?

Reply:
Isn't archive.org the same thing?

Reply:
No, web.archive.org can very easily archive web urls and updating them is super easy. To get something on archive.org you need to manually upload it.

Reply:
That's a good point. I made a separate issue (#132781) for it as it's not related to the PR (src.url was not changed here, just moved up to a let expr)